src/transaction/requests.rs

```diff
@@ -43,9 +32,9 @@ impl KvRequest for MvccGet {
         &mut self,
         pd_client: Arc<PdC>,
     ) -> BoxStream<'static, Result<(Self::KeyData, Store<PdC::KvClient>)>> {
-        let key = self.key.clone();
+        let key = mem::replace(&mut self.key, Vec::default()).into();
```

Comment:
Why we consume the `self.key`? Would it be used again later?

Reply:
Never mind, the one in the original request will not be used again.